bin/build-augmentations.mjs

```diff
@@ -51,7 +51,7 @@ const filePath = resolve(
   '../../packages/template/-private/dsl/lib.dom.augmentation.d.ts',
 );
 let content = prefix;
-content += 'export const GlintSymbol: unique symbol = Symbol();\n\n';
+content += 'export const GlintSymbol: unique symbol;\n\n';
```

Comment:
This is needed because otherwise I get errors due to runtime code (instantiating a Symbol) trying to operate in ambient .d.ts context.